homeassistant/components/vicare/strings.json

```diff
@@ -424,6 +424,35 @@
       },
       "compressor_phase": {
         "name": "Compressor phase"
+      },
+      "ventilation_demand": {
+        "name": "Ventilation demand",
+        "state": {
+          "unknown": "Unknown",
+          "ventilation": "Ventilation"
+        }
+      },
+      "ventilation_level": {
+        "name": "Ventilation level",
+        "state": {
+          "unknown": "Unknown",
```

Comment:
Let's not have `unknown` as valid state, rather have the integration return `None` when that happens, which will translate to `unknown`